elearning-frontend/src/components/admin/Categories.js

```diff
@@ -0,0 +1,272 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import apiClient from '../../services/api'
+
+const Categories = (props) => {
+    
+    const navigate = useNavigate()
+
+    // Cateogry List
+    const [categoryList, setCategoryList] = useState([])
+
+    const [changeData, setChangeData] = useState(true)
+    const [isLoading, setIsLoading] = useState(true)
+
+    // Category
+    const [hasSelectedCategory, setHasSelectedCategory] = useState(false)
+    const [category, setCategory] = useState({
+        id: '',
+        title: '',
+        description: '',
+        slug: '',
+    })
+
+    const usertoken = localStorage.getItem('user')
+
+    // Fetch all categories
+    const FetchCategories = useCallback(async () => {
+        const data = await apiClient({
+            method: "get",
+            url: "/api/admin/get-categories",
```

Comment:
```suggestion
            url: "/api/admin/categories",
```
update all your api request